test/pages/Checkout.page.js

```diff
@@ -0,0 +1,31 @@
+class CheckoutYourInformation {
+   
+
+    get addFirstName() {  
+        return $('#first-name');
+     }
+
+     get addLastName() {  
+        return $('#last-name');
+    }
+    get addPostalCode() {  
+        return $('#postal-code');  
+    }
+
+     async addValueInCheckoutPage(firstname = "firstname", lasttname = "lasttname", code = 123 ) {
```

Comment:
it's better to use object with default values in such case. 
I.e.
```suggestion
     async addValueInCheckoutPage({ firstname = "firstname", lasttname = "lasttname", code = 123}  =  {} ) {
```